src/components/PatchesPage/index.tsx

```diff
@@ -139,15 +136,20 @@ export const getPatchesInputFromURLSearch = (search: string): PatchesInput => {
   };
 };
 
-const FlexRow = styled.div`
-  display: flex;
-  flex-grow: 2;
-`;
 const PaginationRow = styled.div`
   display: flex;
   justify-content: flex-end;
   align-items: center;
 `;
 const FiltersWrapperSpaceBetween = styled(FiltersWrapper)`
-  justify-content: space-between;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr) 2fr;
+  grid-template-rows: 1fr;
+  grid-column-gap: ${size.s};
+  grid-row-gap: 0px;
```

Comment:
I think you can remove the `grid-template-rows` and `grid-row-gap` properties since it seems like they are inferred